test/integration_test.go

```diff
@@ -7510,3 +7510,32 @@ func (ts *IntegrationTestSuite) TestRawValueQueryMetadata() {
 	ts.Equal("Basic", metadata.Definition.Type)
 	ts.Equal(3, len(metadata.Definition.QueryDefinitions))
 }
+
+func (ts *IntegrationTestSuite) TestActivityCancelServer() {
```

Comment:
Nit: I don't think this test name is very clear since you are not testing an activity is canceled by the server